desc/plotting.py

```diff
@@ -4295,10 +4295,10 @@ def plot_gammac(
         Default: 0.5
     alphas : array_like, optional
         Fieldline label values (toroidal angle).
-        Default: np.linspace(0, 2π, 32, endpoint=True)
+        Default: np.linspace(0, 2π, 25, endpoint=True)
     num_pitch : int, optional
         Number of pitch angle values for bounce integral calculation.
-        Default: 16
+        Default: 28
```

Comment:
Can `nufft_eps` be added as a kwarg to the plotting function for gammaC

Reply:
If I make that change it's just going to be replaced with the compute kwargs PR.